feat(website): new documentation (#1048)
algolia/docsearch: packages/website/docs/who-can-apply.md

---
title: Who can apply?
authors:
  - name: Clément Vannicatte
    title: Software Engineer @ Algolia
    url: https://github.com/shortcuts
---

We built DocSearch from the ground up with the idea of improving search on large technical documentations. For this reason, we are offering a free hosting version to all online technical documentations of open-source projects and technical blogs.

We're always sad to have to turn down applications, but with the number of requests we receive every day, we've had to focus on fair criteria. We hope you understand.

## The checklist

To have your request validated, we'll ask that you check all the following points.

> - You must be the **owner** of the website, or at least have the permissions to update its content. You'll have to [include a JavaScript snippet][1] to enable DocSearch.

Reviewer: Not a good suggestion, but it currently seems it's mandatory while it's not
```suggestion
- You must be the **owner** of the website, or at least have the permissions to update its content. You'll need to [include a JavaScript snippet][1] to power your DocSearch frontend.
```

Author: ```suggestion
- You must be the **owner** of the website, or at least have the permissions to update its content. You'll need to [include a JavaScript snippet][1] to your frontend to implement DocSearch.
```

> - Your website must be **publicly available**. We do not host search indices for websites that are available behind an authentication or a private network.

Reviewer: ```suggestion
- Your website must be **publicly available**. We do not host search indices for websites that are only available behind an authentication or a private network.
```

- Your website must be a **technical documentation of an open source project or a technical blog**. We do not index commercial content.

> - Your website must **have some final content**. We won't index empty websites nor those filled with lorem ipsum placeholder content. Please, wait until you have written some documentation before applying. We would be happy to help you as soon as you have a steady design.

Reviewer: Not sure about `final content`, prod ready or something similar might be more clear.

Author: Indeed, [commit](https://github.com/algolia/docsearch/pull/1048/commits/e1399bc2f189aa7b4cb3d13bddcf62ff39287c3f#diff-c685d47ff3a9e15566050cf8ace0c621da322863f00fbf1bda7ea79f87eafa68R23)

If in doubt, don't hesitate to [apply][2] and we'll figure it out together.

> Even if we cannot accept your request, this does not mean that you cannot enjoy great search on your website. DocSearch is entirely open source and [you can run it yourself][3], or use any of [our other API clients][4] to take advantage of Algolia's features.

Reviewer: ```suggestion
Even if we cannot accept your request, this does not mean that you cannot enjoy great search on your website. Legacy DocSearch is entirely open source and [you can run it yourself][3], or use any of [our other API clients][4] to take advantage of Algolia's features.
```

You can also mention that the Crawler is open for trial

title: Who can apply?
---

We built DocSearch from the ground up with the idea of improving search on large technical documentations. For this reason, we are offering a free hosting version to all online technical documentations of open-source projects and technical blogs.

We're always sad to have to turn down applications, but with the number of requests we receive every day, we've had to focus on fair criteria. We hope you understand.

## The checklist

To have your request validated, we'll ask that you check all the following points.

- You must be the **owner** of the website, or at least have the permissions to update its content. You'll need to [include a JavaScript snippet][1] to your frontend to implement DocSearch.

> - Your website must be **publicly available**. We do not host search indices for websites that are only available behind an authentication or a private network.

Reviewer 2: ```suggestion
- Your website must be **publicly available**. We do not host search indices for websites that are only available after authentication or are hosted on a private network.
```

- Your website must be a **technical documentation of an open source project or a technical blog**. We do not index commercial content.

> - Your website must be **production ready**. We won't index empty websites nor those filled with lorem ipsum placeholder content. Please, wait until you have written some documentation before applying. We would be happy to help you as soon as you have a steady design.

Reviewer 2: ```suggestion
- Your website must be **production ready**. We won't index empty websites nor those filled with placeholder content. Please, wait until you have written some documentation before applying. We would be happy to help you as soon as you have a steady design.
```

Maybe we can drop the lorem ipsum reference, I think placeholder content speaks for itself. 😉
